build/config.gypi

```diff
@@ -0,0 +1,80 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
what is this file doing here? this looks like a node-gyp file that should be gitignored in node modules?